.azure-pipelines/scripts/lint.sh

```diff
@@ -7,7 +7,7 @@ set -euo pipefail
 
 ## Formatting
 echo "running gofmt..."
-gofmt_output="$(gofmt -l -s .)"
+gofmt_output="$(gofmt -l -s . | grep -v .azure-pipelines && exit 1 || exit 0)"
```

Comment:
what are these lines doing?

Reply:
This stops the linter giving grief about go files used in the integration tests